Library/Formula/g3log.rb

```diff
@@ -0,0 +1,34 @@
+class G3log < Formula
+  desc "asynchronous, \"crash safe\", logger that is easy to use."
+  homepage "https://github.com/KjellKod/g3log"
+  url "https://github.com/KjellKod/g3log/archive/v1.1.tar.gz"
+  sha256 "b6374819ec9ba56807eeaa6f11e889f33550b1296524a5e7831ce9adc1218a9c"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make"
+    include.install "src/g3log"
+    lib.install Dir["libg3logger*"]
```

Comment:
Sadly no. I will raise it as an issue with the author. For the mean time it works to just copy/install the headers and static/shared lib.